collaborative-circom/src/lib.rs

```diff
@@ -85,6 +88,32 @@ impl std::fmt::Display for MPCProtocol {
     }
 }
 
+/// Collaborative-circom configuration
+#[derive(Debug, Deserialize, Default)]
+pub struct Config {
+    /// Mpc curve to be sued
+    pub curve: Option<MPCCurve>,
+    /// Mpc protocl to bu sued
+    pub protocol: Option<MPCProtocol>,
+    /// Mpc-vm config
+    pub vm: Option<VMConfig>,
+    /// Mpc-compiler config
+    pub compiler: Option<CompilerConfig>,
+    /// Network config
+    pub network: Option<NetworkConfig>,
+}
+
+impl Config {
+    /// Create a new config with given path   
+    pub fn new(path: &str) -> Result<Self, ConfigError> {
+        let config = ConfigBuilder::builder()
+            .add_source(File::with_name(path))
+            .add_source(Environment::with_prefix("conf"))
```

Comment:
Maybe not conf but COCIRCOM_CONF or similar

Reply:
just COCIRCOM (or maybe already COSNARK?)